misc/language-tour-guide/src/content/3-pattern-matching/matching-tuples.mdx

```diff
@@ -0,0 +1,32 @@
+export const frontmatter = { order: 21 };
+
+# Matching Tuples
+
+Thanks to pattern matching, you can "extract" each tuple element into a variable.
+The pattern must be of the same size for the match to succeed.
+
+In Elixir, it is common to return a tuple `{:ok, result}` on success and either `:error` atom
+or `{:error, reason}` tuple in case of failure.
+You can also use atoms in the pattern to match on such results.
+
+<EditorCode>
+
+```elixir
+{x, y, z} = {:hello, "world", 42}
+IO.inspect(x, label: "x")
+IO.inspect(y, label: "y")
+IO.inspect(z, label: "z")
+
+# Uncomment to see the match error
+# {a, b} = {:hello, "world", 42}
```

Comment:
For such simple things that are easy to describe, I'd go for asking to change the code, not just uncommenting